osticket/1.0.0/README.md

```diff
@@ -0,0 +1,47 @@
+  # Keptn osTicket Integration
+
+  osTicket is a popular open source ticketing system. Using the Keptn webhook-service it is possible to create tickets in osTicket from Keptn events.
+
+  ## Step 1: Create osTicket API key
+
+  Go to the osTicket Staff Control Panel and generate an API Key: `https://example.com/scp/apikeys.php`
+
+  Note that for tickets to be created, the IP address of the submitter (ie. Keptn) must be listed in the API key. Give your API key `Can Create Tickets` permission.
+
+  ![osticket api key](assets/1.png)
```

Comment:
As described in https://github.com/keptn-contrib/artifacthub/pull/104 Artifacthub currently doesn't support relative paths. The reason is that when processing packages AH only reads the metadata files. They don't store or serve any other files. Therefore all the relative paths in the PR should be changed to absolute paths. Will also add this fact to the README to avoid future problems.